README.md

```diff
@@ -237,61 +239,46 @@ the output to a log file or other log management system.  See the
 
-1) Unzip it to a directory of your choice on the target system.
+If [Chocolatey](https://chocolatey.org/) is installed on your Windows system, you
+can install the agent by running the following Powershell command as an
+Administrator:
 
-2) Ensure a valid configuration file is available somewhere on the target
-system.  The main thing that the installer script provides -- but that you will
-have to provide manually with the bundle -- is a run directory for the agent to
-use.  Since you aren't installing from a package, there are two config
-options that you will especially want to consider:
-
- - `internalStatusHost` - This is the host name that
-	 the agent will listen on so that the `signalfx-agent status` command can
-	 read diagnostic information from a running agent.  This is also the host name the
-	 agent will listen on to serve internal metrics about the agent.  These metrics can
-	 can be scraped by the `internal-metrics` monitor.  This will default to `localhost`
-	 if left blank.
+```sh
+choco install signalfx-agent --params "'/access_token:YOUR_SIGNALFX_API_TOKEN'"
+```
 
- - `internalStatusPort` - This is the port that the agent will listen on so that
-	 the `signalfx-agent status` command can read diagnostic information from
-	 a running agent.  This is also the host name the agent will listen on to serve
-	 internal metrics about the agent.  These metrics can can be scraped by the
-	 `internal-metrics` monitor.  This will default to `8095`.
+The following package parameters can be set:
 
-See the section on [Privileges](#privileges) for information on the
-capabilities the agent requires.
+ * `/access_token` - The token used to send metric data to SignalFx. If not specified, the value in the `\ProgramData\SignalFxAgent\token` file (if it exists) will be used. See the **note** below for details regarding the token and starting the SignalFx Smart Agent service after installation.
+ * `/ingest_url` - Base URL of the SignalFx ingest server. If not specified, the value found in the `\ProgramData\SignalFxAgent\ingest_url` file (if it exists) will be used. Otherwise, defaults to `https://ingest.signalfx.com`.
+ * `/api_url` - Base URL of the SignalFx API server. If not specified, the value found in the `\ProgramData\SignalFxAgent\api_url` file (if it exists) will be used. Otherwise, defaults to `https://api.signalfx.com`.
+ * `/install_dir` - Installation directory. Defaults to `\Program Files\SignalFx\SignalFxAgent`.
 
-3) Run the agent by invoking the agent executable
-`SignalFxAgent\bin\signalfx-agent.exe -config <path to config.yaml>`.  By default,
-the agent logs only to stdout/err. If you want to persist logs, you must direct
-the output to a log file or other log management system.  See the
-[signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
-supported command flags.
+The agent will look for a configuration file at
+`\ProgramData\SignalFxAgent\agent.yaml`.  If one does not already exist during
```

Comment:
If it doesn't already exist, 
[What copies the sample config file? The install package?][Avoid passive voice at all costs]